src/components/layer-manager/layer-time/layer-time-slider.vue

```diff
@@ -1,21 +1,30 @@
 <script setup lang="ts">
 import { computed } from 'vue'
+import { useTranslation } from 'i18next-vue'
 
 import { formatTimeValue } from '@/services/time.utils'
 import { Layer, LayerTimeMode } from '@/stores/map.store.model'
+import SliderRange from '@/components/common/slider-range/slider-range.vue'
+
+const { t } = useTranslation()
 
 const props = defineProps<{
   layer: Layer
 }>()
 const emit = defineEmits<{
-  (e: 'changeTime', dateStart: string): void
+  (e: 'changeTime', dateStart?: string, dateEnd?: string): void
 }>()
 
-const sliderValue = computed(() =>
+const selectedMinValue = computed(() =>
   props.layer.time?.values?.findIndex(
     val => val === props.layer.currentTimeMinValue
   )
 )
+const selectedMaxValue = computed(() =>
+  props.layer.time?.values?.findIndex(
+    val => val === props.layer.currentTimeMaxValue
+  )
+)
 const timeValues = computed(computeTimeValues)
 const sliderNbSteps = computed(() => timeValues.value?.length)
```

Comment:
Be careful, the original ngeo component could accept unequal intervals for the value list. A each step, the closest value was looked up:
https://github.com/camptocamp/ngeo/blob/27dac4c574ce275956e04b855c48cbdfa45ab832/src/layertree/timeSliderComponent.js#L326

We have to decide if we accept unequal intervals between time values or if we impose equal division, this might have no negative impact on the project.

Reply:
Yes you are right, it should be more a timeline than a step by step component. I kept this point for the last Jira ticket, because I wanted to concentrate my efforts on the rendering only. All in all, I'll make the change in this PR.